system-int-test/test/cmr/system_int_test/search/variable/variable_search_test.clj

```diff
@@ -136,6 +140,44 @@
       [variable1 variable4]
       {:name ["Variable1" "*other"] "options[name][pattern]" true}
 
+      ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+      ;; Alias Param
+      "By alias case sensitive - exact match"
+      [variable1]
+      {:variable-alias "Alias1"}
+
+      "By alias case sensitive, default ignore-case true"
+      [variable1]
+      {:variable-alias "alias1"}
+
+      "By alias ignore case false"
+      []
+      {:variable-alias "alias1" "options[variable-alias][ignore-case]" false}
```

Comment:
Our APIs all use underscores rather than dash case. It's fine if dash case also works but we should have tests verifying the underscore versions as well.

(both `variable_alias` and `options[variable_alias][...]` should be tested).

Reply:
Done.